recipe/meta.yaml

```diff
@@ -52,37 +51,35 @@ requirements:
 
 outputs:
   - name: libexactreal
-{% if build_flavour != 'release' %}
-    skip: True
-{% endif %}
+    skip: True   # [build_flavour != 'release']
     script: install-libexactreal.sh
     requirements:
       build:
         # Add compiler as a build dependency again so we get a proper hash for this package.
         - {{ compiler('cxx') }}
-      run:
-        - {{ pin_compatible('e-antic') }}
+      host:
+        - {{ pin_compatible('e-antic', exact=True) }}
```

Comment:
@isuruf: I don't get why this works. The documentation says that `pin_compatible` looks up the package in the build environment and pins to a compatible version (here the same.) But, where is `e-antic` in the build environment here?

Reply:
From the top level build environment, I think.

Reply:
This doesn't work. See gitter